test/TestProject/TestProject.csproj

```diff
@@ -9,7 +9,6 @@
     <ReadCommittedSnapshot>True</ReadCommittedSnapshot>
     <ServiceBrokerOption>EnableBroker</ServiceBrokerOption>
     <PackageProjectUrl>https://github.com/jmezach/MSBuild.Sdk.SqlProj/</PackageProjectUrl>
-    <GenerateCreateScript>True</GenerateCreateScript>
```

Comment:
Was there a specific reason for moving this out into a separate project? Since we're not building any of the other projects as part of CI currently this feature is not being tested end-to-end in CI (except for the unit tests)

Reply:
@jmezach  - yes.... do now that the artifact is copied to properly, any projects that reference this project (e.g. `TestProjectWithSDKRef`) will bring in not only the .dacpac but also the .sql file - the code in the DacpacTool will then see the .sql file as part of a file to incorporate into a new dacpac (which is wrong) and DacFx produces errors when trying to add this file.

So, I made a new project to test project to exercise just this functionality without breaking anything else as nothing will reference the new project.

Reply:
That might actually be an issue. Perhaps we should ignore any `.sql` files in the `bin` and `obj` folders by default as they could have been generated by the build so we'll probably don't want to include them.

Reply:
Yes, I would definitely agree with this. In my projects at work, we always add `Content Remove` nodes for `bin/` and `obj/`. Not sure why it didn't occur to me to suggest this as a broader change to the framework -- I can't see why we would ever want to include these.